src/main/java/by/exadel/internship/controller/FormController.java

```diff
@@ -18,9 +25,11 @@
 
     private final FormService formService;
 
-    @PostMapping
+    @PostMapping(consumes = {MULTIPART_FORM_DATA_VALUE, APPLICATION_JSON_VALUE})
     @ApiOperation("Add new form")
-    public void addNewForm(@RequestBody FormRegisterDTO form) {
-        formService.saveForm(form);
+    public ResponseEntity<Form> addNewForm(@RequestPart(name = "form") FormRegisterDTO form,
+                                           @RequestPart(name = "file", required = false) MultipartFile file) {
+        Form createdForm = formService.process(form , file);
+        return new ResponseEntity<>(createdForm, HttpStatus.OK);
```

Comment:
@RestController is the combination of @Controller and @ResponseBody. So it's excessive to wrap into ResponseEntity.

```suggestion
        return createdForm;
```

Reply:
done